README.md

```diff
@@ -1,6 +1,6 @@
 # MetaLogger
 
-[![Build Status](https://travis-ci.com/FindHotel/meta_logger.svg?branch=master)](https://travis-ci.com/FindHotel/meta_logger)
+![meta_logger](https://github.com/FindHotel/meta_logger/workflows/meta_logger/badge.svg?branch=master)
 
 Wrapper for Elixir.Logger that keeps logger metadata from caller processes.
```

Comment:
Please add an usage example of the Tesla middleware. Or at least just mention that it exists.